api/src/operations/trigger/index.ts

```diff
@@ -0,0 +1,27 @@
+import { defineOperationApi } from '@directus/shared/utils';
+import { getFlowManager } from '../../flows';
+
+type Options = {
+	flow: string;
+	data: string;
+};
+
+export default defineOperationApi<Options>({
+	id: 'trigger',
+
+	handler: async ({ flow, data }, context) => {
+		const flowManager = getFlowManager();
+
+		const parsedData = JSON.parse(data);
+
+		let result: unknown | unknown[];
+
+		if (Array.isArray(parsedData)) {
+			result = await Promise.all(parsedData.map((data) => flowManager.runOperationFlow(flow, data, context)));
```

Comment:
Should we consider doing this sequentially or adding an option to choose between sequential and parallel? Wondering if there's ever a case where you'd want the second iteration to rely on the results of the first 🤔